setup_wizard/wizard.py

```diff
@@ -58,12 +57,14 @@ def main() -> None:
     with config_file.open("w") as f:
         f.write(out)
 
+    proj_name = os.getenv("MICROPUB_WIZARD_PROJECT_NAME", "micropub")
```

Comment:
The default project name should be `microblogpub`: the default path for the repo will be `microblog.pub` and docker-compose removes the dot in the dir name.

Reply:
The current default is always "app" even though it checks every time there's no other possible setting unless you re-write the wizard dockerfile by hand ahead of time